strawberry/schema/schema.py

```diff
@@ -521,7 +546,9 @@ async def execute(
                         result = execution_context.result
                     # Also set errors on the execution_context so that it's easier
                     # to access in extensions
-                    if result.errors:
+
+                    # TODO: maybe here use the first result from incremental execution if it exists
+                    if isinstance(result, GraphQLExecutionResult) and result.errors:
```

Comment:
not sure what I meant with that comment :D 

but we should probably get rid of the isinstance check?

Reply:
If I remember it correctly, there was one incremental execution type that did not have `errors` in it (not totally sure, it's been a while 😅)

If the idea is just to limit to results that have `errors`, maybe `hasattr(result, "errors")` would be easier?

Reply:
so, I think I remember it now, basically when dealing with an incremental response, we could do `result = result.initial_result`, which could make the code better

something for another PR :D